keylime/tenant.py

```diff
@@ -1542,14 +1542,14 @@ def main(argv=sys.argv):  # pylint: disable=dangerous-default-value
         mytenant.registrar_ip = args.registrar_ip
     if args.registrar_port is not None:
         mytenant.registrar_port = args.registrar_port
-
+    interval = config.getfloat("tenant", "retry_interval")
```

Comment:
Using the retry interval also as a timeout value seems hacky. 
#1068 already adds some timeout to these requests, so you can drop it from this PR.

Reply:
If I drop from this PR, pylint will fail

Reply:
It is probably better to have a hard coded timeout (like 60s) instead of using the retry interval setting for the timeout.

Reply:
Fair enough